doc/releases/migration-guide-3.7.rst

```diff
@@ -123,6 +123,49 @@ Controller Area Network (CAN)
 Display
 =======
 
+* ST7735R based displays now use the MIPI DBI driver class. These displays
```

Comment:
you may also add a release not entry along the lines of "Updated ST7735R based displays to use the MIPI DBI driver class", and of course keep this (great) entry for migration instructions.